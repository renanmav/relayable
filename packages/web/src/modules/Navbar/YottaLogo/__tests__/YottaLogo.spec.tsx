import React from 'react'
import { render } from '@testing-library/react'

import YottaLogoComponent from '../../YottaLogo'

it('render correctly', () => {
  const component = render(<YottaLogoComponent />)

  expect(component.container).toMatchSnapshot()
})
