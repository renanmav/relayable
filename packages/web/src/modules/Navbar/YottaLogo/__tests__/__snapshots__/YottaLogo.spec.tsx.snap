// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div>
  <img
    alt="Relayable logo"
    className="makeStyles-root-1"
    onClick={[Function]}
    src="assets/img/yotta-logo.svg"
  />
</div>
`;
